recipes/lessR/meta.yaml

```diff
@@ -0,0 +1,88 @@
+{% set version = '3.9.0' %}
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-lessr
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url:
+    - {{ cran_mirror }}/src/contrib/lessR_{{ version }}.tar.gz
+    - {{ cran_mirror }}/src/contrib/Archive/lessR/lessR_{{ version }}.tar.gz
+  sha256: 1f787748b2ddd4d0ce411651869f34adf9b2e91463cd4be3b9c03b9a53072aea
+
+build:
+  merge_build_host: True  # [win]
+  number: 0
+  noarch: generic
+
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+requirements:
+  build:
+    - {{posix}}zip               # [win]
+
+  host:
+    - r-base
+    - r-colorspace
+    - r-ellipse
+    - r-foreign
+    - r-lattice
+    - r-latticeextra
+    - r-leaps
+    - r-openxlsx
+    - r-png
+    - r-rmarkdown
+    - r-robustbase
+    - r-sas7bdat
+    - r-triangle
+    - r-viridislite
+    - r-wesanderson
+
+  run:
+    - r-base
+    - r-colorspace
+    - r-ellipse
+    - r-foreign
+    - r-lattice
+    - r-latticeextra
+    - r-leaps
+    - r-openxlsx
+    - r-png
+    - r-rmarkdown
+    - r-robustbase
+    - r-sas7bdat
+    - r-triangle
+    - r-viridislite
+    - r-wesanderson
+
+test:
+  commands:
+    - $R -e "library('lessR')"           # [not win]
+    - "\"%R%\" -e \"library('lessR')\""  # [win]
+
+about:
+  home: https://CRAN.R-project.org/package=lessR
+  license: GPL-2
```

Comment:
We try to use the shorthands from
https://opensource.org/licenses/GPL-2.0

So this would be GPL-2.0